vampire.cpp

```diff
@@ -94,7 +94,11 @@ Problem* preprocessProblem(Problem* prb)
   // and that's the moment we want to set the random seed (no randomness in parsing, for the peace of mind)
   // the main reason being that we want to stay in sync with what profolio mode will do
   // cf ProvingHelper::runVampire
-  Lib::Random::setSeed(env.options->randomSeed());
+  if (env.options->randomSeed() != 0) {
+    Lib::Random::setSeed(env.options->randomSeed());
+  } else {
+    Lib::Random::resetSeed();
```

Comment:
I am not sure I want this. (I was happy that the default was reproducible also when not explicitly specifying any seed.)

Are you sure you need this, @joe-hauns ?

Reply:
I see. Well i was thinking the default should be "actual random", but I'm fine with either. When i run i always set the seed explicitly.
I'm fine with either. We can simply change the default value of `randomSeed()` back to one so that the user will get reproducibility by defaults, but can choose for "actual randome" by setting it to `0`...?

Reply:
OK, a good idea! Done :)